webtestingii/src/App.test.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { render, fireEvent } from "react-testing-library";
+import "jest-dom/extend-expect";
+import App from "./App";
+
+describe("<App />", () => {
+  it("renders without crashing", () => {
+    const div = document.createElement("div");
+    ReactDOM.render(<App />, div);
+    ReactDOM.unmountComponentAtNode(div);
+  });
+
+  it("renders successfully", () => {
+    render(<App />);
+  });
+
+  it("successfully strikes", () => {
+    const { getAllByText, getByText } = render(<App />);
+    const button = getAllByText(/strike/i)[1];
+    fireEvent.click(button);
+    getByText(/1/);
```

Comment:
note that this will pass if _any_ text of "1" is found on the DOM, not just the strike counter--so if you accidentally mix up the strike and ball functions to the wrong buttons, `balls` would show 1 instead of strikes but this would still pass